.travis.yml

```diff
@@ -21,8 +21,6 @@ git:
 
 env:
```

Comment:
Why do we still have the Travis config? Isn't GitHub CI sufficient?

Reply:
Yep, still need it.

For a Nixpkgs cache loophole, explained in: https://github.com/haskell-nix/hnix/issues/682#issuecomment-667540898

Travis currently builds and pushes Cachix Nixpkgs cache that GitHub CI, while secret is not provided.

Reply:
I'm not thrilled to babysit _two_ CI systems! Please get this fixed soon.

Reply:
If someone should not be thrilled to babysit _two_ CI systems - is I. Since I did major work working on them and maintaining them and obvious that I am who would maintain them in the nearest future.

And would say even more. Since it is free tiers and our CIs in the future would have different paradigms (classical and FP, aka Hydra or Hercules or Sorkis CI) -  more CI systems - may be beneficial. Some CI would have good Windows testing, some would provide true FP Nix CI with all its abilities, some other services would provide supplementary functions.

I feel vibes that you may need to rest and relax some. I am the one that need to worry about CIs, not you. I am calm and reserved about all this, I just try to understand your argumentation, so I respond to you in the talk.

For example, see the response: https://github.com/dhall-lang/dhall-haskell/issues/1678#issuecomment-592960057

Sometimes 3-4 CIs are the best stable setup that Free Software Haskell project can do currently.

Reply:
Since Travis CI is supplementary at this point - I would make it optional and a fast skip, until we have normal cache process.

Reply:
Sorry, @Anton-Latukha. It's hard for me stay relaxed about this when I receive an email from Travis CI informing me about the broken `master` branch every single day. Situation like the one in https://github.com/haskell-nix/hnix/pull/680 are also very frustrating to me because the CI failure appears to be completely unrelated to my work and there is no information on how the build could be fixed.

I very much hope that you will put an end to this misery.

Reply:
> I gave you a strong argument why, Travis currently provides the cache for GitHub CI, until John puts the a secret key that only he holds and should hold. So currently there is Travis to produce the Nixpkgs cache that gets shared over Cachix so the GitHub CI does not rebuilds the Nixpkgs stack for 3 hours at every commit someone does in PR. It can not be fixed sooner then John appears and provides the secret key for us, until then all our Nixpkgs builds tied to Travis CI.

Ping @jwiegley.

Reply:
Well, no hurry, I already pinged him a couple of times, it is not urgent. I've could of course directly required this action in advance.

I've just made all Travis builds "optional" and enabled "fast skip" of them, and added a "bypass build" that returns "exit" as soon as instance boots up, so Travis always green and passes in 40 seconds for us, and builds and provides caches in the background.